build/run-local.sh

```diff
@@ -5,7 +5,8 @@ KUBECONFIG=${1-openshift.local.clusterup/openshift-apiserver/admin.kubeconfig}
 echo "Using KUBECONFIG '$KUBECONFIG'"
 
 oc login -u system:admin
-oc project default
```

Comment:
Why change the project name? If anything, make it a parameter of env variable that can be tweaked 😉

Reply:
Thanks, @galderz . Will change to the env var with 'default' project name by default :)

Reply:
Thanks! Looks good to me :)